src/briefcase/bootstraps/toga.py

```diff
@@ -71,6 +71,9 @@ def pyproject_table_linux_system_debian(self):
     "libcairo2-dev",
     # Needed to compile PyGObject wheel
     "libgirepository1.0-dev",
+    # Needed to run the app
+    "gir1.2-gtk-3.0",
+    # "gir1.2-webkit2-4.0",
```

Comment:
minor nit - we should include a comment saying why this is commented out.
```suggestion
    # Needed if your app uses WebView
    # "gir1.2-webkit2-4.0",
```